### PR TITLE
Document GHC-24180

### PR DESCRIPTION
**Pull request by david-christiansen**

### CODE REVIEW

File: message-index/messages/GHC-24180/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Too few arguments to type operator
```
Comment (PPKFS) [inline message-index/messages/GHC-24180/index.md:2]:
```suggestion
title: Too few arguments to infix operator
```

I wasn't quite sure how to suggest this. Either I think it needs to be clear from the title that this is for *binary* operators (and therefore this is an infix problem), or that the summary should suggest there are *multiple* operands required

edit: after reading the examples and realising that this is for *type level operators*, I think the above is still valid but I think some emphasis that this is a *type level problem* rather than a *value level problem*

File: message-index/messages/GHC-24180/no-unicode/index.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  |
+3 | ident :: ∀ α.α → α
+  |          ^
+```
+
+## Explanation
+
+The `UnicodeSyntax` language extension causes GHC's parser to recognize a variety of non-ASCII mathematical symbols as representing their Haskell equivalents. The complete list is available in [the documentation](https://downloads.haskell.org/ghc/latest/docs/html/users_guide/exts/unicode_syntax.html).
+
+When this extension is not enabled, `∀` is instead recognized as an infix operator character that has no argument on its left side.
```
Comment (PPKFS) [inline message-index/messages/GHC-24180/no-unicode/index.md:18]: This seems very strange to be bundled with the above type operator problem, given how much there is a separation of other things by error. But if it is indeed the same error, this is nonblocking on this pr :)

Comment (david-christiansen) [reply message-index/messages/GHC-24180/no-unicode/index.md:18]: This error occurs for users who forget to enable `UnicodeSyntax` and then attempt to write `forall` with `∀`, and I find ti deeply mysterious in that context. I think it's especially useful to document cases where error test is not particularly clearly related to the actual situation.

**Comment by david-christiansen** [comment]
> Alas github (and I guess gitlab too from work reasons) doesn't have "I want this comment to be pointed out because it needs further discussion so I don't want to immediately resolve it, but I don't want it to be blocking to the pR"

I'm not sure what you mean by this one - can you rephrase?

**Comment by PPKFS** [comment]
The point about unicode characters was very interesting, and I think it's excellent that it's covered. *However*, it strikes me as strange that this is the error it throws.

**Comment by david-christiansen** [comment]
I agree that it's a strange error, but documenting strange errors is at least as important as documenting non-strange ones :-)

**Review by PPKFS — APPROVED** [review]
